GBA/IO.zig

```diff
@@ -0,0 +1,95 @@
+//! Access to MMIO registers
+
+const Display = @import("Display.zig");
+const DisplayMode = Display.DisplayMode;
+const DisplayStatus = Display.DisplayStatus;
+const MosaicSettings = Display.MosaicSettings;
+const Bg = @import("background.zig");
+const GBA = @import("core.zig").GBA;
+const Interrupt = GBA.Interrupt;
+const InterruptFlags = GBA.InterruptFlags;
+
+const IO_BASE_ADDR = 0x4000000;
+
+pub const Input = @import("Input.zig");
+
+/// Display Control Register
+/// 
+/// (REG_DISPCNT)
+pub const display_ctrl: *volatile Display.DisplayControl = @ptrFromInt(IO_BASE_ADDR);
```

Comment:
What do you think about `display_control` instead of abbreviating `ctrl` ?

Reply:
Prefer the latter, but not hard set on it. I definitely prefer `ctrl` to `cnt` like in a lot of the GBA documentation online.

I tend to like replacements like `addr` and `ctrl` that are completely ubiquitous, even with code completion it just helps keep single lines from getting too long. I did have to stop myself from making some less common abbreviations though.